src/test/java/apitests/retrofit/TrelloBoardTest.java

```diff
@@ -0,0 +1,168 @@
+package apitests.retrofit;
+
+import apitests.restassured.BoardForTests;
+import okhttp3.OkHttpClient;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import retrofit2.Call;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.testng.Assert.assertEquals;
+
+public class TrelloBoardTest {
+
+    String key;
+    String token;
+    Retrofit retrofit;
+    List<String> idListOfCreatedBoards;
+
+    @BeforeClass
+    public void init(){
+        idListOfCreatedBoards = new ArrayList<>();
+        Properties properties = new Properties();
+        String pathToFile = "src/main/resources/trello.properties";
+
+        try {
+            properties.load(new FileInputStream(pathToFile));
+            key = properties.getProperty("key");
+            token = properties.getProperty("token");
+        }
+        catch (IOException ex){
+            ex.printStackTrace();
+        }
+
+        OkHttpClient.Builder httpClient = new OkHttpClient.Builder();
+        retrofit = new Retrofit.Builder()
+                .baseUrl("https://trello.com")
+                .addConverterFactory(GsonConverterFactory.create())
+                .client(httpClient.build())
+                .build();
+    }
+
+    @AfterClass
+    public void clean() throws IOException {
+        BoardService service = retrofit.create(BoardService.class);
+        Call<BoardForTests> callSync;
+        for (String id : idListOfCreatedBoards){
+            callSync = service.deleteBoard(id, key, token);
+            callSync.execute();
+        }
+    }
+
+    @Test
+    public void creatingNewBoardTest() throws IOException {
+        BoardForTests board = new BoardForTests();
+        board.setName("NewBoard");
+        board.setDesc("This is new board!");
+        board.setClosed(false);
+
+        BoardService service = retrofit.create(BoardService.class);
+        Call<BoardForTests> callSync = service.postBoard(key, token, board);
+
+        Response<BoardForTests> response = callSync.execute();
+        BoardForTests createdBoard = response.body();
+
+        if (createdBoard != null) {
```

Comment:
since it's a test, it's better to use assertion that createdBoard is not null, and add it to the list below the assertion